apps/dashboard/components/date-time-picker.tsx

```
@@ -0,0 +1,81 @@
+      <PopoverContent className="w-auto p-0" {...popoverContentProps}>
+        <Calendar
+          mode="single"
+          selected={selectedDateTime}
+          //@ts-expect-error Calendar can't infer the mode correctly
+          onSelect={(date) => handleSelect(date ?? new Date())}
+          initialFocus
+          {...calendarProps}
+        />
+        <Separator />
+        <div className="p-3 flex flex-col gap-2">
+          <label className="text-sm font-medium leading-none ml-0.5">{timeInputLabel}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)